app/src/main/kotlin/com/wire/android/ui/home/sync/FeatureFlagNotificationViewModel.kt

```diff
@@ -86,7 +86,7 @@ class FeatureFlagNotificationViewModel @Inject constructor(
         viewModelScope.launch {
             coreLogic.getSessionScope(userId).observeFileSharingStatus().collect {
                 if (it.isFileSharingEnabled != null) {
-                    featureFlagState = featureFlagState.copy(isFileSharingEnabledState = it.isFileSharingEnabled!!)
+                    featureFlagState = featureFlagState.copy(isFileSharingEnabledState = false)
```

Comment:
if isFileSharingEnabled is not null we disallow filesharing ? Also why would it be null ? Maybe we can map the nullability to something meaningfull or at least move it into the observeFileSharingStatus() so that is emits false when it is nulll

Reply:
oupss, it was a leftover from testing. Thanks for the sharp eye! 🙏